Dockerfile

```diff
@@ -118,6 +118,7 @@ RUN apk add chromium chromium-chromedriver
 COPY .prettierignore ${APP_HOME}/.prettierignore
 COPY .prettierrc ${APP_HOME}/.prettierrc
 COPY .rspec ${APP_HOME}/.rspec
+COPY .env.test ${APP_HOME}/.env.test
```

Comment:
This shouldn't be necessary. `docker-compose.test.yml` should already hook up the environment variables from that file.